src/main/resources/data/pieces/5e317dc2-deeb-4429-b2a1-91e5cd0fd5f7.json

```diff
@@ -5,8 +5,8 @@
   "format": "Physical",
   "itemId": "522a501a-56b5-48d9-b28a-3a8f02482d97",
   "locationId": "53cf956f-c1df-410b-8bea-27f712cca7c0",
-  "poLineId": "d471d766-8dbb-4609-999a-02681dea6c22",
+  "poLineId": "90fb53f2-cdf1-11e8-a8d5-f2801f1b9fd1",
+  "receivedDate": "2018-10-10T00:00:00.000Z",
```

Comment:
The `receivingStatus` is `Expected` but `receivedDate` is specified...

Reply:
removed `receivedDate` field from sample